xla/service/gpu/transforms/copy_fusion_test.cc

```diff
@@ -533,26 +533,47 @@ TEST_F(CopyFusionTest, FusionFitsInBudget) {
 }
 
 TEST_F(CopyFusionTest, FusionExceedsBudget) {
```

Comment:
Please add a companion test with a fusion that has one less parameter and verify that copy fusion triggers.

Reply:
Let's extract the duplicated part into a helper function.